source/ui/BioXAS/BioXASSSRLMonochromatorConfigurationView.h

```diff
@@ -0,0 +1,116 @@
+#ifndef BIOXASSSRLMONOCHROMATORCONFIGURATIONVIEW_H
+#define BIOXASSSRLMONOCHROMATORCONFIGURATIONVIEW_H
+
+#include <QGroupBox>
+#include <QDoubleSpinBox>
+#include <QComboBox>
+#include <QLayout>
+#include <QLabel>
+
+#include "beamline/BioXAS/BioXASSSRLMonochromator.h"
+#include "ui/BioXAS/BioXASSSRLMonochromatorRegionControlView.h"
+#include "ui/BioXAS/BioXASSSRLMonochromatorRegionControlEditor.h"
+
+#define BRAGG_OFFSET_MIN -1000
+#define BRAGG_OFFSET_MAX 1000
+#define BRAGG_SETTLING_TIME_MIN -1000
+#define BRAGG_SETTLING_TIME_MAX 1000
```

Comment:
Hey @helfrij , normally we don't do this type of `#define` action ... that being said we usually do something way worse and just put magic numbers right in the code. I don't imagine there's one-size-fits-all solution here ... but I bet it's the start of a good conversation. Maybe we can have some guidelines as to when to use `#define` versus member functions versus enums versus magic numbers. 

@dretrex @epengr @iainworkman @XuetaoLiu-CLS @dleacock 

(As it stands I'm fine with this. Everything looks good to me otherwise. I can probably merge if you like ... if I don't jump the gun this time.)